front/app/components/Form/Components/Controls/SentimentLinearScaleControl/index.tsx

```diff
@@ -193,21 +167,66 @@ const SentimentLinearScaleControl = ({
                         <ScreenReaderOnly>
                           {getAriaLabel(visualIndex, maximum)}
                         </ScreenReaderOnly>
-                        <StyledImg
-                          src={getSentimentEmoji(visualIndex)}
-                          alt=""
-                          aria-hidden
-                          className={getClassNameSentimentImage(
-                            data,
-                            visualIndex
+                        <Box>
+                          {visualIndex === data && (
```

Comment:
Found this a bit hard to grasp: could you think of having a better name for data + add a short explainer next to visualIndex's creation (the why of it)?